web/src/pages/index.md

```diff
@@ -29,14 +29,13 @@ The design and development of the smart air hockey table will adhere to the
 following focused design requirements to ensure a successful and captivating
 gameplay experience:
 
-1. PSDR #1 (Hardware): An ability to control a grid of lights beneath the
-   playing surface to create complex visuals.
-2. PSDR #2 (Hardware & Software): An ability to accurately track the puck’s
-   position on the playing field in real time.
-3. PSDR #3 (Hardware): An ability to reliably detect goals scored by monitoring
-   the passage of the puck through the goal.
-4. PSDR #4 (Hardware): An ability to internally track the score of ongoing
-   games, which is then displayed with a display.
-5. PSDR #5 (Software): An ability to display dynamic light effects on the
-   playing surface based on game state by implementing software that synchronizes
-   isual effects with ongoing gameplay.
+1. PSDR #1 (Hardware): 
+An ability to control a grid of RGB LEDs beneath the playing surface with a refresh rate of [TODO].
+2. PSDR #2 (Hardware & Software):
+An ability to track the puck’s position on the playing field with a maximum average latency of [TODO].
+3. PSDR #3 (Hardware):
+An ability to reliably detect goals scored by monitoring the passage of the puck through the goal using a photoresistor and LED pair.
+4. PSDR #4 (Hardware):
+An ability to internally track the score of ongoing games, which is then displayed with an LCD display.
```

Comment:
This one needs to be changed, likely won't implement LCD.

Potential: An ability to display debug information about the table and gameplay on an OLED display connected via [TODO]. (thinking of the STM32 OLED)

Reply:
Now that I think about it, OLED would be better than lCD.
The STM32 OLED uses I2C it looks like.  I can either add I2C if we feel comfortable with that, or I can leave it out and see if they say anything.
Also, do we really want to say we will display debug information about the table and gameplay? I feel like displaying the score would be enough, given how simple their examples have been.

Reply:
I'm saying we put the OLED on the master PCB, and it can either be covered during normal use or not easily readable.  We could also display score if we wanted (toggle between dev/normal modes), but the screen is small so it may not be a good idea to focus on it.

Reply:
Let's do I2C, otherwise it's serial which is fine but you lose 8 data pins

Reply:
If we aren't going to use an OLED for displaying score, then how will we show the score? That's a functionality inherent to an air hockey table

Reply:
On the table after goals?

Reply:
Revised: An ability to display game and debug information about the table on an OLED display connected via the I2C communication protocol.